frontend/src/components/groups/Group.tsx

```diff
@@ -1,103 +1,82 @@
 import React, { useEffect, useState } from "react";
-import { Box, Button, Grid, Link } from "@mui/material";
+import { Box, Button } from "@mui/material";
 import Layout from "../Layout";
 import { RootState } from "../../types/data";
 import { useDispatch, useSelector } from "react-redux";
-import { fetchGroupAbout } from "../../actions/group";
+import { deleteGroup, fetchGroupAbout } from "../../actions/group";
 import { fetchGroupRole } from "../../actions/authorization";
 import Typography from "@mui/material/Typography";
-import { useParams } from "react-router-dom";
+import { useNavigate, useParams } from "react-router-dom";
 import { AuthWrapper } from "../auth/AuthWrapper";
 import PersonAddAlt1Icon from "@mui/icons-material/PersonAddAlt1";
 import MembersTable from "./MembersTable";
-import { EditMenu } from "./EditMenu";
 import AddMemberModal from "./AddMemberModal";
 import RoleChip from "../auth/RoleChip";
-import { MainBreadcrumbs } from "../navigation/BreadCrumb";
-import { ErrorModal } from "../errors/ErrorModal";
-import DeleteGroupModal from "./DeleteGroupModal";
+import DeleteIcon from "@mui/icons-material/Delete";
+import { ActionModal } from "../dialog/ActionModal";
 
 export function Group() {
 	// path parameter
 	const { groupId } = useParams<{ groupId?: string }>();
 
+	// use history hook to redirect/navigate between routes
+	const history = useNavigate();
+
 	// Redux connect equivalent
 	const dispatch = useDispatch();
-
 	const fetchGroupInfo = (groupId: string | undefined) =>
 		dispatch(fetchGroupAbout(groupId));
 	const fetchCurrentGroupRole = (groupId: string | undefined) =>
 		dispatch(fetchGroupRole(groupId));
+	const groupDeleted = (groupId: string | undefined) =>
+		dispatch(deleteGroup(groupId));
 
 	const groupAbout = useSelector((state: RootState) => state.group.about);
 	const role = useSelector((state: RootState) => state.group.role);
-	const groupCreatorEmail = useSelector(
-		(state: RootState) => state.group.about.creator
-	);
-	const adminMode = useSelector((state: RootState) => state.user.adminMode);
 
-	const groupCreatorEmailLink = `mailto:${groupCreatorEmail}`;
 	const [addMemberModalOpen, setAddMemberModalOpen] = useState(false);
 	const [deleteGroupConfirmOpen, setDeleteGroupConfirmOpen] = useState(false);
 
 	// component did mount
 	useEffect(() => {
 		fetchGroupInfo(groupId);
 		fetchCurrentGroupRole(groupId);
-	}, []);
-
-	useEffect(() => {
-		fetchGroupInfo(groupId);
-		fetchCurrentGroupRole(groupId);
-	}, [adminMode]);
-
-	// Error msg dialog
-	const [errorOpen, setErrorOpen] = useState(false);
-
-	// for breadcrumb
-	const paths = [
-		{
-			name: "Groups",
-			url: "/groups",
-		},
-		{
-			name: groupAbout.name,
-			url: `/groups/${groupAbout.name}`,
-		},
-	];
+	}, [groupAbout]);
 
 	return (
 		<Layout>
-			{/*Error Message dialogue*/}
-			<ErrorModal errorOpen={errorOpen} setErrorOpen={setErrorOpen} />
-			{/*breadcrumb*/}
-			<Grid container>
```

Comment:
any reason you are removing the breadcrumb?

Reply:
No, this must have been a mistake. I have put it back in.